src/components/formatters/test/src/SmartFactoryTestHelper.cc

```diff
@@ -457,6 +457,7 @@ std::shared_ptr<ISchemaItem> CSmartFactoryTest::InitStructSchemaItem_Common_1(
   Members root_members_map;
   root_members_map[""] =
       SMember(CObjectSchemaItem::create(struct_members), true);
+  // cppcheck-suppress redundantAssignment
```

Comment:
@Ypostolov Are you sure that exactly `SMember(CObjectSchemaItem::create(schema_members), true);` is redundant assignment? Not `SMember(CObjectSchemaItem::create(struct_members), true);`? In this function we have two variables `Members struct_members;` and `Members schema_members;` and it seems like that one of them is redundant, thus all operation with it are redundant too